tests/unit/latency-monitor.tcl

```diff
@@ -23,6 +21,17 @@ start_server {tags {"latency-monitor needs:latency"}} {
         assert_match {calls 2 histogram_usec *} [latency_histogram set]
     }
 
+    test {LATENCY HISTOGRAM sub commands} {
+        r config resetstat
+        r client id
+        r client list
+        # parent command reply with its sub commands
+        assert {[llength [r latency histogram client]] == 4}
+        assert_match {*client|*client|*} [r latency histogram client]
+        assert_match {calls 1 histogram_usec *} [latency_histogram client|id]
+        assert_match {calls 1 histogram_usec *} [latency_histogram client|list]
```

Comment:
i'd rather also match the full command name in the reply for all 3 of these.

Reply:
also, let's check that `client|id` and `client|list` are correctly named in the reply of the unfiltered `LATENCY HISTOGRAM` (as far as i could tell no other test covers that)